tests/golden/copy-command/copy-a-directory-with-force.gold

```diff
@@ -0,0 +1,7 @@
+/
```

Comment:
Here's a list of things we should also test:
`view`
  - [ ] test a path pointing to a specific entry
  - [ ] test a path pointing to a specific entry with a field filter

`create`
  - [ ] `create` fails if an entry already exists at that path
  - [ ] `create --force` succeeds if an entry already exists at that path
  - [ ] specifying the same tag multiple times is the same as specifying it once
  - [ ] field contents can be empty
  - [ ] field contents can have multiple lines

`tag`
  - [ ] fails when the path doesn't exist

`set-field`
  - [ ] can update an existing field's contents
  - [ ] fails when the path doesn't exist

`find`
  - [ ] sort by a field's value (asc/desc)
  - [ ] sort by a field's date (asc/desc)
  - [ ] check that `coffer find PATH` excludes entries that are not in `PATH`
  - [ ] check that `coffer find` returns the intersection (and not the union) of all filters
  - [ ] check that `coffer find PATH TEXT` searches for `TEXT` in the entry's path segments (including the entry name) and tags
  - [ ] check that directories where no matches were found do not appear in the result
  - [ ] check that PATH can be an entry's path
  - [ ] check that using a path that doesn't exist returns no matches
  - [ ] test `--filter-field`

`delete-field`
  - [ ] Fails when the path doesn't exist
  - [ ] Fails when the path is a directory

Reply:
Have written all these tests, except for `field contents can have multiple lines` in `create` section. I don't know how to create a field with multiple lines in its contents.

Reply:
> Have written all these tests, except for field contents can have multiple lines in create section. I don't know how to create a field with multiple lines in its contents.

You can do something like this:
```
coffer create /path --field "user=$(print "first\nsecond" )"
```

Reply:
Done